### ISSUE TITLE
scriptscanner scan value

### ISSUE THREAD

**Issue by jayich** [CampbellGroup/common#59]
scriptscanner should display the current value that is being scanned.

**Comment by aransfor** [comment]
@jayich just to clarify you mean display the dynamic data or the scan? As I see it the scan is already displayed however since the scripter determines how the scan is done Im not sure how to implement some live view of the data

**Comment by jayich** [comment]
I don't remember.

If you scan parameter x from 1 to 3 in 3 steps does scriptscanner display 1, 2, and 3 as they are being scanned?

**Comment by aransfor** [comment]
@jayich I dont think it can because the person who writes the run function in the experiment class controls the iteration, for instance they could do nothing or as is sometimes the case iterate many times at one point. I suppose the answer here would be if you  really wanted it to display you would connect to parameter vault in your experiment and update the value programatically in your script, this would update the tree view value live as you say.

**Comment by jayich** [comment]
What if you scan a parameter with script scanner?  Perhaps that is the case I'm thinking of.

**Comment by aransfor** [comment]
Ah I see perhaps this should have some file naming convention (which I thought it did) I can delve deeper into this
